src/Avalonia.Themes.Default/ButtonSpinner.xaml

```diff
@@ -46,12 +46,15 @@
                 BorderBrush="{TemplateBinding BorderBrush}"
                 BorderThickness="{TemplateBinding BorderThickness}"
                 Margin="{TemplateBinding Padding}"
-                HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-                VerticalAlignment="{TemplateBinding VerticalContentAlignment}">
+                HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                VerticalAlignment="{TemplateBinding VerticalAlignment}">
           <Grid ColumnDefinitions="*,Auto">
             <ContentPresenter Name="PART_ContentPresenter" Grid.Column="0"
                               ContentTemplate="{TemplateBinding ContentTemplate}"
-                              Content="{TemplateBinding Content}"/>
+                              Content="{TemplateBinding Content}"
+                              HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
```

Comment:
Are both `HorizontalAlignment` and `HorizontalContentAlignment` needed here?

Reply:
Let me check that but it should be possible to align the button itself and the content of the button. Will make some tests. The Problem of most these templates was the fact that the ContenAlignment wasn't used properly.

Reply:
Just checked the unchanged Version and i can confirm that both are needed. The current template has some wrong bindings. VerticalContentAlignment and HorizontalContentAlignment should only effect the content of the control the unchanged version applies the Alignment to the Control itself. So if you want to set HorizontalContentAlignment="Center" you dont center the content but center the control itself.